goldstone/core/models.py

```diff
@@ -2696,6 +2701,69 @@ def build_alert_template(self, hits):
         return msg_params_dict
 
 
+class AlertSearchSQLQuery(AlertSearch):
+
+    db_table = models.CharField(max_length=64,
```

Comment:
Why do we need this additional field.  We discussed using the index-prefix field as the table of interest, and refactoring the name to be more generic later.